src/map/utils/synthutils.cpp

```diff
@@ -546,19 +538,35 @@ namespace synthutils
                 penalty += 1;
             }
 
-            if (PChar->CraftContainer->getQuantity(0) == SYNTHESIS_FAIL) // If synth breaks, lower skill up rate
-            {
-                penalty += 1;
-            }
+            // No proof this is a thing. Until proven otherwise removing.
+            // if (PChar->CraftContainer->getQuantity(0) == SYNTHESIS_FAIL) // If synth breaks, lower skill up rate
+            // {
+            //     penalty += 1;
+            // }
 
-            skillUpChance = skillUpChance / penalty; // Lower skill up chance if synth breaks
+            // https://www.bluegartr.com/threads/57123-Before-you-ask-a-stupid-crafting-question-read-this!?p=1987222&viewfull=1#post1987222
+            skillUpChance = skillUpChance / penalty; // Lower skill up chance if desyntth
 
             // Section 3: Calculate Skill Up and Skill Up Amount
             double random = xirand::GetRandomNumber(1.);
 
             if (random < skillUpChance) // If character skills up
             {
                 uint8 skillUpAmount = 1;
+                uint8 maxSkillUp    = 4; // Max skill is 0.4
```

Comment:
https://www.bluegartr.com/threads/57123-Before-you-ask-a-stupid-crafting-question-read-this!?p=1987222&viewfull=1#post1987222
"Below 60, skillup increments can be 0.1 or 0.2.
If your natural skill is 6* or more below the recipe cap, skillup increments can also be 0.3.
If your natural skill is 12** or more below the recipe cap, skillup increments can also be 0.4"

Implies 1-5 gap can only get up to 0.2. The logic following will not catch 1-5 gaps under 60, so setting maxskillup to 4 here would allow 4 gaps to get 0.3 and 5 gaps 0.4 with lucky rolls.

Modified version of the code, but you can see basediff is 5 and maxskillup is 5:
![image](https://user-images.githubusercontent.com/118334503/203472368-7b27d537-940b-4653-8e6f-2ae7327fea50.png)